docker-compose.yml

```diff
@@ -7,7 +7,10 @@ services:
     build:
       context: .
       dockerfile: docker_utils/Dockerfile
-    command: [sh, -c, "medallion --host 0.0.0.0 docker_config.json"]
+    command: [sh, -c, "medallion --host 0.0.0.0"]
+    env:
+      MEDALLION_BACKEND_MODULE_CLASS: "MongoBackend"
```

Comment:
I do wish this could just be `MEDALLION_BACKEND: "MongoBackend"` but `environ-config` doesn't allow us to do variable name customisation like that :(